src/zenml/integrations/bentoml/model_deployers/bentoml_model_deployer.py

```diff
@@ -455,3 +457,27 @@ def delete_model_server(
             self._clean_up_existing_service(
                 existing_service=service, timeout=timeout, force=force
             )
+
+    def get_step_run_metadata(
```

Comment:
Oh nice if this logic is the same for all deployer flavors we can even move it into the base deployer class 👍

Reply:
moved it here 👍